sourcecode/apis/contentauthor/app/Libraries/H5P/Framework.php

```diff
@@ -1335,4 +1351,27 @@ public function libraryHasUpgrade($library)
         $h5pLibrary = H5PLibrary::fromLibrary($library)->first();
         return !is_null($h5pLibrary) && $h5pLibrary->isUpgradable();
     }
+
+    public function replaceContentHubMetadataCache($metadata, $lang)
+    {
+        // H5P Content Hub is not in use
+    }
+
+    public function getContentHubMetadataCache($lang = 'en')
+    {
+        // H5P Content Hub is not in use, but return value to make PHPStan happy
```

Comment:
"Making phpstan happy" is a bad reason to do something. These methods simply have to be implemented to satisfy the requirements of the interface.